src/common/transak-helper.ts

```diff
@@ -8,5 +8,7 @@ const SUBDOMAIN = IS_PRODUCTION ? 'global' : 'staging-global';
 const SCREEN_TITLE = 'Buy Stacks';
 
 export function transakUrl(address: string | undefined) {
```

Comment:
The name of this function is a noun. Can we make it a verb like `makeTransakUrl`?